### PR TITLE
docs(core): Added description of benchmark

### PR DESCRIPTION
**Pull request by EdamAme-x**
Added description of benchmark

### CODE REVIEW

File: .vitepress/config/en.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@ export const enConfig = defineLocaleConfig("root", {
       "/blog/": BLOG_SIDEBAR,
     },
     editLink: {
-      pattern: "https://github.com/oxc-project/oxc/edit/main/src/:path",
+      pattern: "https://github.com/oxc-project/oxc-project.github.io/edit/main/src/:path",
```
Comment (NozomuIkuta) [inline .vitepress/config/en.ts:122]: Would you merge `main` branch and remove this diff?

File: .vitepress/config/ja.ts [hunk 2]
```diff
@@ -155,7 +155,7 @@ export const jaConfig = defineLocaleConfig("ja", {
       })),
     },
     editLink: {
-      pattern: "https://github.com/oxc-project/oxc/edit/main/src/:path",
+      pattern: "https://github.com/oxc-project/oxc-project.github.io/edit/main/src/:path",
```
Comment (NozomuIkuta) [inline .vitepress/config/ja.ts:158]: Would you merge `main` branch and remove this diff?

**Review by NozomuIkuta — CHANGES_REQUESTED** [review]

**Comment by EdamAme-x** [comment]
ok

**Comment by EdamAme-x** [comment]
https://github.com/oxc-project/oxc-project.github.io/pull/56

**Pull request closed without merging**